ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py

```diff
@@ -43,3 +43,36 @@ def test_paddle_layer_norm(
         bias=offset[0],
         epsilon=eps,
     )
+
+
+# instance_norm
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.instance_norm",
+    values_tuple=_generate_data_layer_norm(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
We should test for all "valid" data types, by using the keywork 'valid' here instead of 'float'